R/plotBoxplot.R

```diff
@@ -796,7 +1004,7 @@ setMethod("plotBoxplot", signature = c(object = "SummarizedExperiment"),
 
 # This function adds points to plot
 .add_points_layer <- function(
-        p, df, point.alpha = 0.65, point.size = 2, point.shape = 21,
+        p, df, point.alpha = 0.65, point.size = 2, point.shape = 19L,
```

Comment:
Default point shape is changed. This does not have fill, so it won't be colored with `fill.by` (like boxplot does)